### PR TITLE
[OUM-14] [BE]- Create User registration function

### PR DESCRIPTION
**Pull request by Thaweesak-sai**

### CODE REVIEW

**Review by aicquewk — APPROVED** [review]
lgtm

File: backend/src/account/account.service.ts [hunk 1]
```diff
@@ -2,39 +2,32 @@ import { Injectable } from '@nestjs/common';
 import { InjectModel } from '@nestjs/mongoose';
 import { Model } from 'mongoose';
 import { UnExpectedServerException } from '../common/exceptions/un-expected.server.exception';
```
Comment (yee2542) [inline backend/src/account/account.service.ts:4]: Un-use import

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: backend/src/account/account.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  constructor(private accountService: AccountService) {}
+
+  @UseFilters(UserMongoErrorFilterException)
+  @Mutation(() => UserResponseDTO)
+  async createAccount(
+    @Args('UserRegisterInputDTO') userRegisterInputDTO: UserRegisterInputDTO,
+  ): Promise<UserResponseDTO> {
+    const user = await this.accountService.createAccount(userRegisterInputDTO);
+    return user as UserResponseDTO;
+  }
```
Comment (yee2542) [inline backend/src/account/account.resolver.ts:20]: need test for this resolver

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: backend/src/account/account.resolver.ts (hunk 2 above)
Comment (yee2542) [reply backend/src/account/account.resolver.ts:20]: seem it fixed on c1f5ced2326822653756a8aa5ade14b1551720c0

**Review by yee2542 — APPROVED** [review]

**Pull request merged**